### PR TITLE
fix(Textbox): Add missing labelAlign prop to textbox.d.ts

### PR DESCRIPTION
**Pull request by gregor-gh**
### Proposed behaviour
Add a missing labelAlign property on the Textbox d.ts file. 

### Current behaviour
The property is missing at the moment so attempting to use throws a typescript error
Property 'labelAlign' does not exist on type 'IntrinsicAttributes & TextboxProps & RefAttributes<HTMLInputElement>'.

### Checklist

- [x] Commits follow our style guide
<del>- [] Screenshots are included in the PR if useful<del>
<del>- [] All themes are supported if required<del>
<del>- [] Unit tests added or updated if required<del>
<del>- [] Cypress automation tests added or updated if required<del>
<del>- [] Storybook added or updated if required<del>
- [x] Typescript `d.ts` file added or updated if required
<del>- [] Carbon implementation and Design System documentation are congruent<del>

### Additional context
n/a

### Testing instructions
Attempt to use the labelAlign prop on a Textbox in a .tsx file.

### CODE REVIEW

**Review by ianforster-sage — CHANGES_REQUESTED** [review]
Hi @gregor-gh, thanks for submitting your first PR to the Carbon repository. I noticed one or two areas that don't follow our guides, please could you correct these before we review it fully.

- your Git commit message doesn't follow our standard https://github.com/Sage/carbon/blob/master/CONTRIBUTING.md#git-commit-messages
- your PR should remain in Draft mode until after it's been peer reviewed
- your PR description should follow our template https://github.com/Sage/carbon/blob/master/.github/PULL_REQUEST_TEMPLATE.md

Thanks again for your contribution, I'll watch the PR and get it reviewed once you've updated it. You can find our contributing guide here https://github.com/Sage/carbon/blob/master/CONTRIBUTING.md

**Comment by gregor-gh** [comment]
Thanks Ian, I think I've updated everything. Was just a quick change I made my side that I figured I'd raise a PR for, wasn't reading too closely.

**Review by ianforster-sage — CHANGES_REQUESTED** [review]
Hey @gregor-gh thanks for the update. Our pipeline makes GPG verification mandatory for all contributors. You can see how to configure that here https://github.com/Sage/carbon/blob/master/docs/dev-environment-setup.md#gpg-verification
Please could you sign your commit and resubmit it then I can run the checks

**Comment by gregor-gh** [comment]
Hi Ian, I've amended the commit with a GPG key as requested.

**Review by ianforster-sage — APPROVED** [review]

**Review by mmaciorowski-sage — APPROVED** [review]

**Review by sjohnson-sage — CHANGES_REQUESTED** [review]
We should add this to the `propTypes` in `textbox.component.js` as well so they match up

**Comment by gregor-gh** [comment]
I've updated textbox.component.js to include labelAlign, and I've also gone back and amended textbox.d.ts to use OptionsHelper.AlignBinaryTypes, rather than explicitly state "left" | "right". "Size" was already specifying small/med/large so I've pointed that at OptionsHelper.SizesRestricted while I was in there; it now matches its textbox.component.js counterpart.

File: src/__experimental__/components/textbox/textbox.d.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import * as React from "react";
 import { SpaceProps } from "styled-system";
 import { ValidationPropTypes } from "../../../components/validations";
+import * as OptionsHelper from "../../../utils/helpers/options-helper";
```
Comment (ianforster-sage) [inline src/__experimental__/components/textbox/textbox.d.ts:4]: OptionsHelper is soon to be removed by this PR https://github.com/Sage/carbon/pull/4134
I recommend that we delay merging this PR until #4134 is merged, then we'll advise on next steps for you

**Review by ianforster-sage — CHANGES_REQUESTED** [review]

**Comment by dawidzarzycki** [comment]
Hey @gregor-gh 
OptionsHelper has been removed in carbon version 79.0.0
We are no longer supporting for OptionsHelper and all types we do, should come from their own files.
Also storybook doesn't work with objects well. We should change values that come from object OptionsHelper with real values.
In your example it will be 
changing this
```js
labelAlign: PropTypes.oneOf(OptionsHelper.alignBinary),
```
to this
```js
labelAlign: PropTypes.oneOf(["left", "right"])
```

Also your d.ts files should be updated with real values instead of these that come from OptionsHelper

This is a link to PR with OptionsHelper
https://github.com/Sage/carbon/releases/tag/v79.0.0

**Comment by dawidzarzycki** [comment]
labelAlign has been added to the textbox.d.ts file 
https://github.com/Sage/carbon/blob/master/src/components/textbox/textbox.d.ts#L35

I think this PR is no longer valid

**Pull request closed without merging**